doc/devices/aer.rst

```diff
@@ -33,33 +33,6 @@ You can then execute the circuit like any other function to get the quantum mech
 
     circuit(0.2, 0.1, 0.3)
 
-Backends
-~~~~~~~~
-
-The default backend is the ``AerSimulator``. However, multiple other backends are also available.
-To get a current overview what backends are available you can query
-
-.. code-block:: python
-
-    dev.capabilities()['backend']
-
-or, alternatively,
-
-.. code-block:: python
-
-    from qiskit_aer import Aer
-    Aer.backends()
-
-.. note::
```

Comment:
This I'm pretty sure doesn't work. Should get some kind of replacement.

Reply:
This version seem so still work for me:
 
```
 from qiskit_aer import Aer
 Aer.backends()
```